ios/RNIterableAPI/Serialization.swift

```diff
@@ -143,8 +143,9 @@ extension IterableHtmlInAppContent {
         var dict = [AnyHashable: Any]()
         dict["type"] = type.rawValue
         dict["edgeInsets"] = edgeInsets.toDict()
-        dict["backgroundAlpha"] = backgroundAlpha
         dict["html"] = html
+        dict["shouldAnimate"] = shouldAnimate
+        dict["backgroundColor"] = backgroundColor ?? nil
```

Comment:
```suggestion
```

Reply:
Yeah, basically if we don't deserialize or use these fields on Javascript side, we don't need to serialize them here.

Reply:
And if we _want_ to make these fields available on Javascript side, we'll need to deseralize and expose them on JS side as well.